iModelCore/ECDb/ECDb/SchemaWriter.cpp

```diff
@@ -134,6 +134,150 @@ DropSchemaResult SchemaWriter::DropSchema(Utf8StringCR schemaName, SchemaImportC
     }
     return DropSchemaResult(DropSchemaResult::Status::Success);
 }
+
+DropSchemaResult SchemaWriter::DropSchemas(T_Utf8StringVectorCR schemaNames, SchemaImportContext& ctx, bool logIssue)

```

Comment:
For clarity, it's advisable to use `bvector<Utf8String>`, which is straightforward and readable. `T_Utf8StringVector` hails from a time before `Utf8String` and `bvector` were standardized as derivatives of `std::string` and `std::vector`, respectively.

Reply:
Done !